### ISSUE TITLE
add missing conda dependencies for aiida-core 1.0b

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida_core#2592]
We're almost there, there are just two packages missing:

```
$ conda env create -f environment.yml python=3.6
Collecting package metadata: done
Solving environment: failed

ResolvePackageNotFound:
  - circus==0.15.0
  - django==1.11.20
```

 * circus is still at 0.14: https://anaconda.org/conda-forge/circus
 * django 1.11.20 is not available

@astamminger Could I ask your help again for the django package?
I can take care of pinging the circus people

**Comment by astamminger** [comment]
Well, I was about to volunteer for the Django stuff anyway :D

**Comment by ltalirz** [comment]
circus 15.0 is now available on all platforms except for windows
only django missing...

**Comment by astamminger** [comment]
I'll try to get to it tomorrow

**Comment by astamminger** [comment]
@ltalirz , django-1.11.20 should now be available (conda-forge/django-feedstock#70)
However, there was an issue with Windows for Django as well which I have not resolved yet. So for now django-1.11.20 will **not** be available for python 2.7 on Windows (3.6/3.7 are just fine and I hope to get this fixed in the next few days, but since circus is not available for Windows either this shouldn't be a big issue)

**Comment by astamminger** [comment]
@ltalirz the Python 2.7 build on Windows for `django-1.11.20` is now fixed.
